pom.xml

```diff
@@ -185,7 +185,7 @@
     <spring.batch.version>3.0.7.RELEASE</spring.batch.version>
     <hibernate.version>5.1.0.Final</hibernate.version>
     <mysql.connector.version>5.1.38</mysql.connector.version>
-    <duracloud.version>4.4.0</duracloud.version>
+    <duracloud.version>4.5.0-SNAPSHOT</duracloud.version>
```

Comment:
Do we need to be using the snapshot version here? Has something changed since 4.4.6 that is required by this update?

Reply:
If we don't want snapshots to fail when no description is provided, this commit is required - https://github.com/duracloud/duracloud/commit/7ac18bc18f95d8ca35e2a0a9a2c55c8e9da7cb1f